index.js

```diff
@@ -3,53 +3,433 @@ const state = {
     {
       id: "006-bananas",
       name: "bananas",
-      price: 0.35
+      price: 0.10,
+      quantity: 0,
+      type: "Fruit"
     },
     {
       id: "007-bell-pepper",
       name: "bell pepper",
-      price: 0.35
+      price: 0.10,
+      quantity: 0,
+      type: "Fake Fruit"
     },
     {
       id: "008-berry",
       name: "berry",
-      price: 0.35
+      price: 0.05,
+      quantity: 0,
+      type: "Fruit"
     },
     {
       id: "009-blueberry",
       name: "blueberry",
-      price: 0.35
+      price: 0.50,
+      quantity: 0,
+      type: "Fruit"
     },
     {
       id: "010-eggplant",
       name: "eggplant",
-      price: 0.35
+      price: 0.40,
+      quantity: 0,
+      type: "Fake Fruit"
     }
   ],
   cart: []
-};
\ No newline at end of file
+};
+const store = document.querySelector('.store--item-list')
+const total = document.querySelector('.total-number')
+const options = document.querySelector('#fruit')
+const order = document.querySelector('#order')
+
+order.addEventListener('input', () => {
+  if (order.value === "a_z") {
+    aToZ()
+      if (options.value === "All") {
+    storeListRender()
+  }
+  if (options.value === "Fruit") {
+    storeListFruitRender()
+  }
+  if (options.value === "Veg") {
+    storeListVegRender()
+  }
+  if (options.value === "Fake") {
+    storeListFakeRender()
+  }
+  }
+  if (order.value === "z_a") {
+    zToA()
+      if (options.value === "All") {
+    storeListRender()
+  }
+  if (options.value === "Fruit") {
+    storeListFruitRender()
+  }
+  if (options.value === "Veg") {
+    storeListVegRender()
+  }
+  if (options.value === "Fake") {
+    storeListFakeRender()
+  }
+  }
+  if (order.value === "highToLow") {
+    highToLow()
+      if (options.value === "All") {
+    storeListRender()
+  }
+  if (options.value === "Fruit") {
+    storeListFruitRender()
+  }
+  if (options.value === "Veg") {
+    storeListVegRender()
+  }
+  if (options.value === "Fake") {
+    storeListFakeRender()
+  }
+  }
+  if (order.value === "lowToHigh") {
+    lowToHigh()
+    if (options.value === "All") {
+      storeListRender()
+    }
+    if (options.value === "Fruit") {
+      storeListFruitRender()
+    }
+    if (options.value === "Veg") {
+      storeListVegRender()
+    }
+    if (options.value === "Fake") {
+      storeListFakeRender()
+    }
+  }
+})
+
+options.addEventListener('input', () => {
+  if (options.value === "All") {
+    storeListRender()
+  }
+  if (options.value === "Fruit") {
+    storeListFruitRender()
+  }
+  if (options.value === "Veg") {
+    storeListVegRender()
+  }
+  if (options.value === "Fake") {
+    storeListFakeRender()
+  }
+})
+
+const totalCost = () => {
+  let allAdded = 0
+  let totalPence = state.cart.forEach( (obj) => {
+    allAdded += obj.price * 100 * obj.quantity
+  })
+  let pounds = Math.floor(allAdded / 100)
+  let pence =  allAdded % 100
+  if (pence < 10) {
+    pence = '0' + pence
+  }
+  total.innerText = `£${pounds}.${pence}`
```

Comment:
I did that because since i had the total in pence e.g 350p by doing math.floor, whatever you divide by it tells you the answer by the nearest integer and disregards the rest so for 350 it would be 3, then i did 350 modulo by 100 so each 100 will be thrown away and leaves me with 50, so then the innerText will be 3.50